misk-hibernate/src/main/kotlin/misk/hibernate/JdbcExtensions.kt

```diff
@@ -0,0 +1,15 @@
+package misk.hibernate
+
+import com.google.common.collect.Iterables.getOnlyElement
+import java.sql.ResultSet
+
+fun <T> ResultSet.map(function: (ResultSet) -> T): List<T> {
```

Comment:
cute

Reply:
Should this be `internal` ? I’m reluctant to publish APIs on JDBC types

Reply:
Can't be internal alas :-( because it's also used in the misk-hibernate-testing module (which has the same package but different module so can't access internal :-( )

Reply:
Gotcha.